lib/components/inputs/TagsBox/tagsBox.scss

```diff
@@ -150,6 +150,21 @@
      color: var(--focus-key);
     font-size: 12px;
   }
+
+  &.expand-input-on-focus {
+    .react-tagsbox__value-container,
+    .react-tagsbox__single-value {
+      max-height: 31px;
+      transition: max-height 0.3s ease;
```

Comment:
transition is nice, but it should be consistent throughout all expandable inputs - either all of them should have it or none of them

Reply:
I'm not familiar with the entire expandable inputs, but transition improve the quality and the look and feel, hence I think that we should use it once changing height on the fly.

Reply:
expandable components are Select and CutomizableSelect - the ones from which you took reference for expanding

Reply:
ok, so I'll add transition for those too